pkg/jobrunaggregator/jobrunbigqueryloader/types.go

```diff
@@ -105,11 +110,20 @@ func (v *testRunRow) Save() (map[string]bigquery.Value, string, error) {
 		status = "Passed"
 	}
 
+	var testSuite_str string
```

Comment:
This was the first inserter I ever made.  Waht do you think about making this a struct like the other "normal" ones now?

Reply:
ah, you mean like how you did [JobRunRow](https://github.com/openshift/ci-tools/blob/master/pkg/jobrunaggregator/jobrunaggregatorapi/types_row_jobrun.go#L57)?